src/services/assetInfo/assetInfo.ts

```diff
@@ -0,0 +1,75 @@
+import { JsonRpcProvider, ZeroAddress } from 'ethers'
+
+import { Network, NetworkId } from '../../interfaces/network'
+import { GetOptions, Portfolio } from '../../libs/portfolio'
+
+const scheduledActions: {
+  [network in NetworkId]?: {
+    promise: Promise<any>
+    data: { callback: Function; address: string }[]
+  }
+} = {}
+
+export async function executeBatchedFetch(network: Network): Promise<any> {
+  const provider = new JsonRpcProvider(network.rpcUrls[0])
+  const allAddresses =
+    Array.from(new Set(scheduledActions[network.id]?.data.map((i) => i.address))) || []
+  const portfolio = new Portfolio(fetch as any, provider, network)
+  const options: Partial<GetOptions> = {
+    disableAutoDiscovery: true,
+    previousHints: {
+      erc20s: allAddresses,
+      erc721s: Object.fromEntries(
+        allAddresses.map((i) => [
+          i,
+          {
+            tokens: ['1'],
+            isKnown: false
+          }
```

Comment:
Can you explain this part?

Reply:
sentinel value, used for single nft fetch from the portfolio